src/psyclone/parse/kernel.py

```diff
@@ -814,6 +814,9 @@ def __init__(self, ast, name=None):
         self._name = name
         self._ast = ast
         self._ktype = get_kernel_metadata(name, ast)
+        # TODO #1204 since the valid form of the metadata beyond this point
```

Comment:
Thanks for adding the "TODO" here. I think the Copyright year should be changed to 2021.

Reply:
Done.